### DISCUSSION TITLE
📎 Create TRANSLATIONS.md, a docs and web translation contribution guideline

### DISCUSSION THREAD

**Discussion by ImBIOS** [biomejs/biome#1177] [Polls]
### Description

At first, I found Portuguese, Japanese, Chinese, and English. I need to add translation in several languages, due to most people living in the world aren't fluent in other than English. If we can create the 15 most spoken languages in the world, we can reach almost 95% of the population in the world and almost 100% of all internet users.

I was inspired by the translation contribution guidelines for [t3-oss/create-t3-app](https://github.com/t3-oss/create-t3-app/blob/main/www/TRANSLATIONS.md). I also read the web translation a bit messy, where the default language `en` is not placed in its folder, but the translations are placed in its folder. if I compare it with t3-oss, the below issue might be related to this problem:

- https://github.com/biomejs/biome/issues/865

I was thinking of directly creating an issue, but I read it's recommended to ask in discussion first. So, here I am. If any privileged user accepts this proposal, you can just convert this discussion directly into an actionable issue.

**Comment by David-Else** [comment]
GPT4 seems very good at translations, and the fact the docs change all the time means that translations need to be constantly updated.

The ultimate system would be a GitHub action is run when any document is updated. GPT is asked to translate into multiple languages (with a temperate of 0 it might avoid it being different every time). Each document would need to be edited and pull requests created for each language, then a native language speaker could be asked to approve it. It could probably all be automated with something like langchain or https://openinterpreter.com/, but that would be a much bigger job. 

As an example of a prompt with GPT4TurboPreview:

```
aichat "You are a technical writer creating software documentation. Translate 'hello my good friend' into French and Japanese, dont add any provisions and don't name the language"
Bonjour mon bon ami.

こんにちは、私の良い友達。
```

https://github.com/sigoden/aichat can be controlled using a simple bash script.

I am afraid I am really busy, but I just thought I would share the idea while it was in my brain.

**Reply by ImBIOS** [reply]
I love 🥰  automation and efficiency, we have more time for the holiday... I mean, for more important thing 😅 .
